colour/io/luts/common.py

```diff
@@ -1,9 +1,8 @@
 # -*- coding: utf-8 -*-
 """
-LUT Processing Common Utilities
-===============================
-
-Defines LUT Processing common utilities objects that don't fall in any specific
+A Common Utility for LUT Processing
+====================================
+Defines a LUT Processing common utility object that doesn't fall in any specific
 category.
 """
```

Comment:
There might be more utilities here as you go-on, let's keep it plural for now if you don't mind please! :)

Reply:
Ok sure haha, I didn't think that way. Will make that change! Thankyou!